### DISCUSSION TITLE
PocketHost Sveltekit Adapter

### DISCUSSION THREAD

**Discussion by rnbokade** [pockethost/pockethost#267] [Q&A]
Since PH supports static files and deno worker functions. How about creating a sveltekit adapter for PH such that the generated static assets are put in static folder and generated backend logic routes are packed into deno functions and shipped into the workers folder

**Comment by benallfree** [comment]
This sounds awesome, is it something you can work on?

**Comment by rnbokade** [comment]
Will try my best to work on it. Will post a link to repository here once I am ready to setup

**Reply by benallfree** [reply]
Great! You'll probably need assistance with the Deno backend. Does Svelte even have a Deno adapter?

**Reply by rnbokade** [reply]
https://github.com/dbushell/sveltekit-adapter-deno/

**Reply by benallfree** [reply]
That looks like an excellent starting point, combined with the static asset adapter. This would be an amazing contribution to PocketHost!
